ports/geckolib/gecko_bindings/bindings.rs

```diff
@@ -276,7 +276,7 @@ extern "C" {
     pub fn Servo_StyleSheetHasRules(sheet: *mut RawServoStyleSheet) -> bool;
     pub fn Servo_InitStyleSet() -> *mut RawServoStyleSet;
     pub fn Servo_DropStyleSet(set: *mut RawServoStyleSet);
-    pub fn Servo_ParseStyleAttribute(bytes: *const u8, length: u32,
```

Comment:
I neglected to cherry pick the patch onto the gecko-dev stylo branch. I'll remember to do that next time.